documentation/asciidoc/computers/raspberry-pi/power-button.adoc

```diff
@@ -0,0 +1,28 @@
+== Power button
+
+NOTE: This section only applies to Raspberry Pi models with a power button, such as the Raspberry Pi 5.
+
+When you plug your Raspberry Pi into power for the first time, it will automatically turn on and boot into the operating system without having to push the button.
+
+If you run Raspberry Pi Desktop, you can initiate a clean shutdown by briefly pressing the power button. A menu will appear asking whether you want to shutdown, reboot, or logout.
```

Comment:
Strictly speaking, you get a window rather than a menu. Not sure it's worth a change, though.

Reply:
Fixed! TY.